pkg/csi-common/server.go

```diff
@@ -109,9 +112,11 @@ func (s *nonBlockingGRPCServer) serve(endpoint string, ids csi.IdentityServer, c
 	if ns != nil {
 		csi.RegisterNodeServer(server, ns)
 	}
-
 	klog.Infof("Listening for connections on address: %#v", listener.Addr())
-
+	if metrics {
+		grpc_prometheus.EnableHandlingTimeHistogram()
```

Comment:
I would like to see if we can enable a different histogram than the default. The default puts all requests that are beyond 10s into one bucket, I think we can be better served if we shift the buckets upto 30 or 60 seconds.

Also, if we can do the above, we can have lower count of buckets than the default of 12 (if memory serves right). Thus reducing number of metric points that are collected and stored across the plugin.

Buckets like, 0.5, 1.0, 2.5, 5.0, 10.0, 25.0, 60.0

Of course at some point this may need to be configurable, but I would think we can achieve that in a later PR when the need arises.

Reference:
[1] Exponential buckets: https://godoc.org/github.com/prometheus/client_golang/prometheus#ExponentialBuckets
[2] Ability to configure buckets in gRPC prom integration (as far as I understand it): https://github.com/grpc-ecosystem/go-grpc-prometheus/blob/ae0d8660c5f2108ca70a3776dbe0fb53cf79f1da/server_metrics.go#L60-L64

Reply:
done

Reply:
I think this may come back for a different default, but let's see. The current 0.5,2,6 will make buckets like, 0.5 - 2 - 8- 32... the workload sensitivity may demand a more granular set of initial buckets than the 2 - 8 - 32 jump.

I would have preferred a non-exponential set in this lower order, say 0.5, 2.0, 5.0, 10.0, 30.0, 60.0 (even though I was walling for an exponential initially).

Leaving this thought here for the future.

Reply:
this is configurable, I will check how to expose this flag in rook.  having non-exponential values need code changes